packages/flutter/test/material/text_form_field_test.dart

```diff
@@ -114,4 +114,28 @@ void main() {
     await tester.pump();
     expect(_validateCalled, 2);
   });
+
+  testWidgets('validate is not called if widget is disabled', (WidgetTester tester) async {
+    int _validateCalled = 0;
+
+    await tester.pumpWidget(
+      MaterialApp(
+        home: Material(
+          child: Center(
+            child: TextFormField(
+              enabled: false,
+              autovalidate: true,
+              validator: (String value) { _validateCalled++; return null; },
+            ),
+          ),
+        ),
+      ),
+    );
+
+    expect(_validateCalled, 0);
+    await tester.showKeyboard(find.byType(TextField));
+    await tester.enterText(find.byType(TextField), 'a');
+    await tester.pump();
+    expect(_validateCalled, 0);
```

Comment:
we should also check that it does validate if you now enable it.

we also need a lot more tests here, verifying the things we care about (e.g. whether selection works when disabled, whether it renders disabled, whether it's focusable, etc -- basically, all the behaviours you want to make sure we don't regress).

Reply:
@Hixie, is there a test already where I can see how it can be verified?

Reply:
never mind, i misread the patch. no further tests are needed.